frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -0,0 +1,303 @@
+                                type="monotone"
+                                dataKey="rate"
+                                stroke="var(--teal-3)"
+                                strokeWidth={2}
+                                name="Completion Rate"
+                            />
+                        </LineChart>
+                    </ResponsiveContainer>
+                </div>
+            </div>
+
+            <div className="flex flex-wrap items-center justify-between gap-2 mb-4">
+                <div className="flex flex-row gap-x-2">
+                    <SearchBar
+                        searchTerm={searchTerm}
+                        changeCallback={handleChange}
+                    />
+                    <DropdownControl
+                        label="order by"
+                        setState={setSortQuery}
+                        enumType={{
+                            'Name (A-Z)': 'name_last asc',
+                            'Name (Z-A)': 'name_last desc',
+                            'Section Created (Newest)': 'created_at desc',
+                            'Section Created (Oldest)': 'created_at asc'
+                        }}
+                    />
+                    {/* pill  */}
+                    <button
+                        className={`px-4 py-2 rounded-full border ${
+                            includeActive
+                                ? 'bg-teal-600 text-white'
+                                : 'bg-white text-teal-600'
+                        }`}
+                        onClick={() => setIncludeActive(!includeActive)}
+                    >
+                        {includeActive
+                            ? 'Active Sessions'
+                            : 'Inactive Sessions'}
+                    </button>
+                </div>
+
+                <button className="button flex items-center">
+                    <PlusCircleIcon className="w-4 h-4 mr-1" />
+                    Add Section
+                </button>
+            </div>
+
+            {/* sections table */}
+            <div className="card p-4">
+                <table className="table w-full mb-4">
+                    <thead className="bg-background">
+                        <tr>
+                            <th>
+                                <input
+                                    type="checkbox"
+                                    checked={allSelected}
+                                    onChange={(e) =>
+                                        handleToggleAll(e.target.checked)
+                                    }
+                                />
+                            </th>
+                            <th>Facility Name</th>
+                            <th>Facilitator</th>
+                            <th>Start Date</th>
+                            <th>End Date</th>
+                            <th>Enrolled Residents</th>
+                            <th>Capacity</th>
```

Comment:
Do you think we could turn this column into a single column that includes:
- A horizontal bar for each section to show the percentage filled.
- Numerical values to the right of the bar (e.g., "15/30") to provide clarity on the enrollment number vs. capacity.